qless/profile.py

```diff
@@ -19,14 +19,14 @@ def pretty(timings, label):
         '''Print timing stats'''
         results = [(sum(values), len(values), key)
             for key, values in timings.items()]
-        print label
-        print '=' * 65
-        print '%20s => %13s | %8s | %13s' % (
-            'Command', 'Average', '# Calls', 'Total time')
-        print '-' * 65
+        print(label)
```

Comment:
Go ahead and add the `from __future__ import print_function` to make the behavior consistent across both 2 & 3.